crypto/asn1/bio_asn1.c

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <internal/bio.h>
 #include <openssl/asn1.h>
+#include "e_os.h"
```

Comment:
The concern about "e_os.h" and also the int cast issue are addressed in #4073. I'll rebase this PR once that one has gone in.